### PR TITLE
Ft badge

### PR DESCRIPTION
**Pull request by BenIraa**
## What does this Pr do?
1. Badge Added [Code Climate, Build, Maintanability, Code Cavellars] 
2. README Having Full Information Of PHANTOM BE

## Description of Task to be completed?

> update README File And
`.yml files`

##How should this be manually tested?

https://github.com/atlp-rwanda/Avengers-Phantom-BE

### CODE REVIEW

**Review by Anguandia — CHANGES_REQUESTED** [review]
Good work @BenIraa, please action the following:
- your PR has no description, please give it a description as indicated in [this](https://github.com/atlp-rwanda/engineering-playbook/tree/master/5.%20Developing/Conventions#branch-naming) best practice guide
- adding badges is not a feature but a chore, rename the branch both locally and on Github by following [this](https://linuxize.com/post/how-to-rename-local-and-remote-git-branch/)
- there is no badge for test coverage, please we need that too
- make use of [this](https://macpaw.com/how-to/remove-ds-store-files-on-mac) to deal with .DS_Store files on your mac
- remove it from version control by adding it to .gitignore by following these [stapes](https://betterprogramming.pub/how-to-remove-committed-files-from-git-version-control-b6533b8f9044)

**Review by kareraolivier — CHANGES_REQUESTED** [review]
Your pr have conflicts that needs to be resolved @BenIraa

File: src/Passenger/Routes/passenger.route.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const { 
+    createPassenger,
+    removePassenger,
+    getAllPassengers,
+    getPassenger,
+    UpdatePassenger,
+} = require("./../Controllers/passenger.controller");
+
+const router = express.Router();
+
```
Comment (capitale250) [inline src/Passenger/Routes/passenger.route.js:14]: these routes has no protection you need to add it

**Review by capitale250 — COMMENTED** [review]
@BenIraa this pr has no full discription and makes it hard to review and test it and resolve conflicts

**Pull request merged**